modules/rpc/yarpc_options.go

```diff
@@ -71,3 +73,20 @@ func onewayInboundMiddlewareFromCreateInfo(mci service.ModuleCreateInfo) []middl
 	// Intentionally panic if programmer adds non-middleware slice to the data
 	return items.([]middleware.OnewayInbound)
 }
+
+func withGraph(graph dig.Graph) modules.Option {
```

Comment:
wait a minute. Why is the graph passed as an object? I thought the whole point of having DI is to perform dependency injection and not pass around as a dependency baggage. These objects should just be access d as 'dig.Resolve()'. Otherwise Im having hard time understanding the use of graph vs passing a struct or an interface map.